.editorconfig

```diff
@@ -0,0 +1,30 @@
+# http://editorconfig.org
```

Comment:
I think we might want to remove it to avoid two source of truth for some things.